README.md

```diff
@@ -1,8 +1,103 @@
+</div>
 
-## Quick Start ##
-Quick guide to run UpGrade locally with Docker: https://upgrade-platform.gitbook.io/docs/developer-guide/usage-guide
+UpGrade is an open-source platform designed to support large-scale A/B testing in any web-application. This combined repository provides tools and resources to integrate A/B testing frameworks into various software solutions. Learn more at www.upgradeplatform.org.
+
+## Features
+
+- 🆎 **A/B Testing**: Implement controlled trials within software for impactful learning analysis.
+- 🚩 **Feature Flagging**: Provision for experiment based feature flagging.
+- 📊 **Segmentation**: Segmentation for inclusion/exclusion of groups.
+- ➿ **Factorial Experiments**: Support for factorial experiment.
+- ♊ **Stratified Random Sampling**: Provision for stratifications for categorical study.
+- ⏫ **Scalability**: Designed to support large-scale experiments.
+- 🔃 **Integration**: Easily integrates with various applications.
+- 💻 **SDKs**: We have SDKs for [Java](https://github.com/CarnegieLearningWeb/UpGrade/tree/dev/clientlibs/java), [Javascript](https://github.com/CarnegieLearningWeb/UpGrade/tree/dev/clientlibs/js) and [C#](https://github.com/CarnegieLearningWeb/UpGrade/pull/860). More to come soon!
+
+    ![Java](https://img.shields.io/badge/java-%23ED8B00.svg?style=for-the-badge&logo=openjdk&logoColor=white) &nbsp;&nbsp; ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E) &nbsp;&nbsp; ![C#](https://img.shields.io/badge/c%23-%23239120.svg?style=for-the-badge&logo=c-sharp&logoColor=white)
+
+https://github.com/CarnegieLearningWeb/UpGrade/assets/33730817/86d72930-9fab-4949-a2ba-3e480a3cdf7e
+
+## Documentation
+
+Find detailed documentation for UpGrade at [UpGrade Documentation](https://upgrade-platform.gitbook.io/docs/).
+
+## Quick Start (Using Docker)
+
+##### Get `Upgrade` running using Nix:
+
+Get started with UpGrade using Docker with this [Quick Start Guide](https://upgrade-platform.gitbook.io/docs/developer-guide/usage-guide) using Nix for separate containers for frontend, backend and postgres DB as separate containers.
+
+<img src="https://img.shields.io/badge/Built_With-Nix-5277C3.svg?logo=nixos&labelColor=73C3D5" />
+
+```markdown
+Get NIX shell to delegate some dependency magic: 
+
+$ sh <(curl -L https://nixos.org/nix/install)
+
+Get the code from https://github.com/CarnegieLearningWeb/UpGrade (monorepo contains all packages):
+
+$ git clone https://github.com/CarnegieLearningWeb/UpGrade.git
+
+Navigate into root of project and run these commands to install dependencies:
+
+$ cd Upgrade
+$ make setup-local
+```
+
+```
+NOTE: If this make command says nix not found and you are using zsh on mac, it may need to be manually added to PATH. OR, just try running it in bash
+```
```

Comment:
This NOTE text appears in a horizontal scroll, rather than wrapping. I think it probably doesn't need to be inside the ```'s.